extensions/wikia/FlowTracking/FlowTracking.hooks.php

```diff
@@ -41,7 +47,7 @@ public static function onArticleInsertComplete( Page $page, User $user, $text, $
 			Track::event( 'trackingevent', [
 				'ga_action' => 'flow-end',
 				'editor' => static::getEditor( $request->getValues(), $queryParams ),
-				'flowname' => $queryParams[ 'flow' ] ?? static::CREATE_PAGE_UNRECOGNIZED_FLOW,
+				'flowname' => $queryParams[ 'flow' ] ?? static::FLOWS[ 'CREATE_PAGE_UNRECOGNIZED_FLOW' ],
```

Comment:
It seems awkward to me, as the whole point is to use const instead of a string value. Why not a const?